snakefile.py

```diff
@@ -193,11 +202,34 @@ def tool(name):
     shell: "{SAMTOOLS_EXEC} index {input} {output} >> {log} 2>&1"
 
 
-rule fastqc:
-    input: os.path.join(MAPPED_READS_DIR, '{sample}_aligned_sorted.bam')
-    output: os.path.join(FASTQC_DIR, '{sample}_aligned_sorted_fastqc.zip')
-    log: os.path.join(LOG_DIR, 'fastqc_{sample}.log')
-    shell: "{FASTQC_EXEC} -o {FASTQC_DIR} -f bam {input} >> {log} 2>&1"
+rule fastqc_raw:
+    input: os.path.join(READS_DIR, "{sample}_R{read_num}.fastq")
+    output: os.path.join(FASTQC_DIR, '{sample}', '{sample}_R{read_num}_fastqc.zip')
```

Comment:
hint for the future: let's use just one type of quote for strings :)

My preference is to use double quotes throughout.

Reply:
yes, I see :)
(unfortunately mine is single quotes... should change this. I know double quotes make more sense at some point)

Reply:
Either way is fine as far as Python is concerned.

My preference comes from Scheme where `'` is used for symbols and `"` is used for strings.  But .... it doesn't matter.  We should just pick one and go with it.